examples/tutorials/run_v230_all_sets.py

```diff
@@ -0,0 +1,95 @@
+import os
+from acme_diags.run import runner
+from acme_diags.parameter.core_parameter import CoreParameter
+from acme_diags.parameter.area_mean_time_series_parameter import AreaMeanTimeSeriesParameter
+from acme_diags.parameter.enso_diags_parameter import EnsoDiagsParameter
+from acme_diags.parameter.qbo_parameter import QboParameter
+from acme_diags.parameter.diurnal_cycle_parameter import DiurnalCycleParameter
+from acme_diags.parameter.streamflow_parameter import StreamflowParameter
+
```

Comment:
I would list all the `prefix` variables up at the top commented with `# Cori` in this case, as in https://github.com/E3SM-Project/e3sm_diags/blob/master/examples/tutorials/tutorial_2020_all_sets.py#L12. That way, tutorial users know what machine the script is for AND they can easily change all the paths in one place if they're using a different machine.